core/src/main/java/com/graphhopper/routing/RoundTripAltAlgorithm.java

```diff
@@ -209,6 +212,17 @@ public void setWeightLimit( double weightLimit )
     }
 
     @Override
+    public void setMaxVisitedNodes( int numberOfNodes )
+    {
+        this.maxVisitedNodes = numberOfNodes;
+    }
+
+    protected boolean isMaxVisitedNodesExceeded()
+    {
+        return maxVisitedNodes > getVisitedNodes();
```

Comment:
Ah yes, we can actually delete the isMaxVisitedNodesExceeded method, since it is not used here. We delegate the maxVisitedNodes to the Dijkstra and the Dijkstra cares about the maxVisistedNodes.